src/services/UsersService.ts

```diff
@@ -1,4 +1,7 @@
 import api from 'config/api';
-import { User } from 'utils/types';
+import { User, UserCredentials } from 'utils/types';
 
 export const signUp = (userPayload: User) => api.post('/users', userPayload);
+
+export const login = (userCredentialsPayload: UserCredentials) =>
+  api.post('/users/sign_in', userCredentialsPayload).then(res => res.headers && res.headers['access-token']);
```

Comment:
If I'm not wrong, it should work like this, making the change that I mark above

@Juliocho990  here can you use only this
```suggestion
  export const login = (userCredentialsPayload: UserCredentials) =>
  api.post('/users/sign_in', userCredentialsPayload);
```

Reply:
I understand what you mean, but I used the chain just to trha to get one property as response
```Typescript
.then(res => res.headers && res.headers['access-token'])
```
It was to try to get just an `access-token` property from headers because at that time (related to the trello card) we just needed to handle one property. 
Then, In the following trello cards I made a little bit changes to get a better response and handle other properties like: `access-token`, `client`, and `uid`.
Let me know if it makes sense to you.

Reply:
I understand, but for me the logic of saving that information in the localStorage should be in the component

Reply:
Ok, I finally got it. 
So, what you say is just keep the response in the service and any kind of logic in the component. I was thinking about it and make sense to me. 
Thanks for this clarification.